charts/umbrella/README.md

```diff
@@ -471,9 +471,19 @@ helm install -f values-adopter-portal.yaml umbrella . --namespace umbrella --cre
 
 Involved components:
 
-EDC, MIW, DTR, Vault (data provider and consumer in tx-data-provider), CentralIdP.
+CentralIdP, DBRS-Server-Memory, Data Consumer, Data Provider, SSI Credential Issuer, SSI Dim Wallet Stub.
```

Comment:
should reflect the list of enabled components [here](https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/values-adopter-data-exchange.yaml)

Could you please add such a list also for the in the get to know the Portal section, with an information about the disabled interfaces https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/values-adopter-portal.yaml#L53?